docs/src/man/basics.md

```diff
@@ -0,0 +1,1505 @@
+```
+
+```jldoctest dataframe
+julia> german[:, Between(:Sex, :Housing)]
+1000×3 DataFrame
+  Row │ Sex     Job    Housing
+      │ String  Int64  String
+──────┼────────────────────────
+    1 │ male        2  own
+    2 │ female      2  own
+    3 │ male        1  own
+    4 │ male        2  free
+    5 │ male        2  free
+    6 │ male        1  free
+    7 │ male        2  own
+    8 │ male        3  rent
+  ⋮   │   ⋮       ⋮       ⋮
+  994 │ male        3  own
+  995 │ male        2  own
+  996 │ female      1  own
+  997 │ male        3  own
+  998 │ male        2  own
+  999 │ male        2  free
+ 1000 │ male        2  own
+               985 rows omitted
+
+julia> german[:, Cols("Age", Between("Sex", "Job"))]
+1000×3 DataFrame
+  Row │ Age    Sex     Job
+      │ Int64  String  Int64
+──────┼──────────────────────
+    1 │     1  male        2
+    2 │     1  female      2
+    3 │    49  male        1
+    4 │    45  male        2
+    5 │    53  male        2
+    6 │    35  male        1
+    7 │    53  male        2
+    8 │    35  male        3
+  ⋮   │   ⋮      ⋮       ⋮
+  994 │    30  male        3
+  995 │    50  male        2
+  996 │    31  female      1
+  997 │    40  male        3
+  998 │    38  male        2
+  999 │    23  male        2
+ 1000 │    27  male        2
+             985 rows omitted
+
+julia> german[:, Cols("Age", Not("Sex"))]
+1000×9 DataFrame
+  Row │ Age    id     Job    Housing  Saving accounts  Checking account  Credi ⋯
+      │ Int64  Int64  Int64  String   String           String            Int64 ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     1      1      2  own      NA               little                  ⋯
+    2 │     1      1      2  own      little           moderate
+    3 │    49      2      1  own      little           NA
+    4 │    45      3      2  free     little           little
+    5 │    53      4      2  free     little           little                  ⋯
+    6 │    35      5      1  free     NA               NA
+    7 │    53      6      2  own      quite rich       NA
+    8 │    35      7      3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮            ⋮                ⋮                ⋱
+  994 │    30    993      3  own      little           little                  ⋯
+  995 │    50    994      2  own      NA               NA
+  996 │    31    995      1  own      little           NA
+  997 │    40    996      3  own      little           little
+  998 │    38    997      2  own      little           NA                      ⋯
+  999 │    23    998      2  free     little           little
+ 1000 │    27    999      2  own      moderate         moderate
+                                                  3 columns and 985 rows omitted
+
+julia> findall(==(67), german[!, :Age]) # It will find 67 in column `Age` of `german` data frame
```

Comment:
this is a different topic and unrelated to the issues above. Maybe add some more comment here?

Reply:
Can you guide, please? I am not getting further. :(

Reply:
you are giving an example of using `findall` on a vector - so this is a topic for Julia Base manual (i.e. https://docs.julialang.org/en/v1/) and not DataFrames.jl manual. We can add it if you show how this relates to DataFrames.jl functionality - otherwise we should assume that readers learn how to use `findall` from Julia Base manual.